src/cancel-reminders/lambda/lambda.ts

```diff
@@ -4,15 +4,12 @@ import * as SSM from 'aws-sdk/clients/ssm';
 import { Pool } from 'pg';
 import { createDatabaseConnectionPool } from '../../lib/db';
 import { getHandler } from '../../lib/handler';
-import { getIdentityIdByEmail } from '../../lib/identity';
```

Comment:
hadn't even thought about the fact that now we don't need an identity lookup! 🚀

Reply:
Yes, in that respect it's simplified things. It does however mean we have to lookup the reminder_code in both signup tables to find the identity_id

Reply:
ahh yeah of course, swings and roundabouts